app/uk/gov/hmrc/gform/email/EmailCodeVerificationController.scala

```diff
@@ -37,12 +37,16 @@ class EmailCodeVerificationController(
 
   def sendEmail() =
     Action.async(parse.json[ConfirmationCodeWithEmailService]) { implicit request =>
-      val ConfirmationCodeWithEmailService(notifierEmailAddress, code, emailVerifierService) = request.body
+      val ConfirmationCodeWithEmailService(notifierEmailAddress, code, emailVerifierService, lang) = request.body
       emailVerifierService match {
-        case EmailVerifierService.Notify(emailTemplateId) =>
+        case EmailVerifierService.Notify(emailTemplateId, maybeEmailTemplateIdCy) =>
+          val templateId = lang match {
+            case LangADT.En => emailTemplateId
+            case LangADT.Cy => maybeEmailTemplateIdCy.getOrElse(emailTemplateId)
+          }
           val notifierEmail: NotifierEmail =
             NotifierEmail(
-              emailTemplateId,
+              templateId,
```

Comment:
Can we not use the `Notify.notifierTemplateId(l: LangADT): NotifierTemplateId` method here?

Reply:
Sure we can, thanks 👍